### PR TITLE
Ppp dg classdiag

### PR DESCRIPTION
**Pull request by KZQ1999**

### CODE REVIEW

**Review by Nirmaleshwar — COMMENTED** [review]
Looks good to merge, maybe can include a class diagram later if you want.

File: src/main/java/seedu/utility/StonksGraph.java [hunk 1]
```diff
@@ -285,5 +286,26 @@ private void drawReport(FinancialTracker finances) {
         }
     }
 
-
+    private double determineBarValue(double max) {
+        double barValue;
+        if (max < 1) {
```
Comment (kyun99) [inline src/main/java/seedu/utility/StonksGraph.java:291]: might want to refactor this part by replacing the conditionals with a method to make the code more readable

File: src/main/java/seedu/utility/StonksGraph.java [hunk 2]
```diff
@@ -285,5 +286,26 @@ private void drawReport(FinancialTracker finances) {
...
+            if (max >= 0.1) {
```
Comment (kyun99) [inline src/main/java/seedu/utility/StonksGraph.java:292]: same comment for this part as well. Would it be better to use a descriptive name to understand what the max comparison does?

**Review by kyun99 — COMMENTED** [review]
Overall code looks good and text seems coherent. Might want to consider the comments in the pr but other than that all else looks good to merge! :)

**Review by kyun99 — APPROVED** [review]
Looks good to merge

**Pull request merged**